build.gradle

```diff
@@ -23,10 +35,14 @@ dependencies {
     compile localGroovy()
     compile ('net.wasdev.wlp.ant:wlp-anttasks:1.4-SNAPSHOT')
     compile group: 'commons-io', name: 'commons-io', version: '2.5'
+    compile gradleApi()
+    compile localGroovy()
     provided group: 'com.ibm.websphere.appserver.api', name: 'com.ibm.websphere.appserver.spi.kernel.embeddable', version: '1.0.0'
     testCompile 'junit:junit:4.11'
 }
 
+apply plugin: "com.gradle.plugin-publish"
```

Comment:
should this be up with the other plugins?

Reply:
Moved to the top with the other plugins.